integ-test/src/test/java/org/opensearch/sql/calcite/remote/CalciteArrayFunctionIT.java

```diff
@@ -241,4 +241,96 @@ public void testReduceWithUDF() throws IOException {
 
     verifyDataRows(actual, rows(60));
   }
+
+  @Test
+  public void testMvjoinWithStringArray() throws IOException {
+    JSONObject actual =
+        executeQuery(
+            String.format(
+                "source=%s | eval result = mvjoin(array('a', 'b', 'c'), ',') | fields result | head"
```

Comment:
Do we need IT for real field with array value?

Reply:
yes adding this now